test/e2e/test.utils.js

```diff
@@ -15,7 +15,8 @@ export const accountAddresses = {
 };
 
 export const EMERYNET_FAUCET_URL = 'https://emerynet.faucet.agoric.net';
-export const DEFAULT_TIMEOUT = 2 * 60 * 1000;
+export const DEFAULT_TIMEOUT = 3 * 60 * 1000;
+export const DEFAULT_TASK_TIMEOUT = 3 * 60 * 1000;
```

Comment:
Curious why distinguishing these now, given they're the same value.

Reply:
It's mainly to ensure consistency and stability across our tests. The `DEFAULT_TIMEOUT` is the timeout for actions involving waiting for DOM elements. The `DEFAULT_TASK_TIMEOUT` applies to custom commands created in `support.js`.

In many tests, we use a mix of both(Wait for DOM elements as well as running custom commands). So increasing the timeout for both, we can achieve more consistent and stable test results. I've implemented similar timeouts in `dapp-inter` for liquidation tests, and they have been very stable with emerynet.

Also, thanks for asking. I just noticed that the tests using both of these timeouts actually need to use `DEFAULT_TASK_TIMEOUT`.

Reply:
Helpful explanation. Please include it in comments in the code.

Reply:
Updated :)